src/features/playback-controls/ui/controls.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { play, pause, seek, move, audio } from 'src/entities/audio/model/audio-instance';
```

Comment:
suggestion: I encourage you to keep FSD layers in the `src/lib` directory. For example, `src/lib/entities/audio/...`. The reason is two-fold:
* That's what Max and I were doing in the other PRs
* That allows importing with the SvelteKit-native alias, `$lib`, without having to define our own aliases

issue: in terms of FSD, code outside of a slice is prohibited to go inside that slice's structure. Every slice should have an `index.ts` defining its public interface, and other slices should stop there:
```suggestion
  import { play, pause, seek, move, audio } from '$lib/entities/audio';
```